http/data-url.json

```diff
@@ -0,0 +1,63 @@
+{
+  "version": "1.0.0",
+  "data": {
+    "http": {
+      "data-url": {
+        "__compat": {
+          "basic_support": {
+            "desc": "data URL scheme",
+            "support": {
+              "webview_android": {
+                "version_added": true
+              },
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": "12",
+                "notes": ["IE9 and later, as well as Edge, supports data URLs in CSS and JS files, but not in HTML files, with a max size of 4GB."]
```

Comment:
I find this note hard to read :).

First, I have tried to avoid referring to browsers in the note itself, because it ends up reading weirdly (why is the Edge data talking about IE?).

Second, the complex run-on sentence makes it ambiguous and hard to parse. It could be read as: `data` is supported in CSS, JS, HTML. In CSS and JS the max size is 4GB.

It would be better written as: "data URLs can only be used in CSS and JS files. The maximum size supported is 4GB."